examples/plot_bg_cube_model_comparison.py

```diff
@@ -17,11 +17,18 @@
 GRAPH_DEBUG = 0
 SAVE = 0
 
+NORMALIZE = 0 # normalize 1 w.r.t. 2 (i.e. true w.r.t. reco)
+              # 0: do not normalize
+              # 1: normalize w.r.t. cube integral
+              # 2: normalize w.r.t images integral (normalize each image on its own)
+
 input_dir1 = '/home/mapaz/astropy/working_dir/gammapy_scripts/20150819_ready_to_merge_PR319/bg_cube_models_gammapy_a_la_michi'
```

Comment:
Hard-coded PATHs make it hard and unlikely that others will run the example.
So it would be much better if you make this a command line option.
For quick scripts it's OK to just use `input_dir = sys.argv[1]` ... no need to use an `ArgumentParser`, so it's just one or two extra lines to change this.